webapp/channels/src/components/no_script/no_script_template.html

```diff
@@ -0,0 +1,17 @@
+<noscript
+    style="
+        background-color: #ffffff;
+        width: 100vw;
+        position: absolute;
+        z-index: 101;
+        font-size: 14px;
+        height: 40px;
+        display: flex;
+        justify-content: center;
+        flex-wrap: nowrap;
+        align-items: center;
+        font-family: sans-serif;
+    "
+>
+To use Mattermost, please enable JavaScript. Here are the &nbsp;<a href="https://www.enable-javascript.com/" rel="noopener noreferrer" target="_blank">instructions on how to enable JavaScript in your web browser</a>.
```

Comment:
I like the idea behind giving more information here, but I don't think we should be directing users to a third party site for this, particularly one that isn't controlled by a major tech company

Reply:
Yes i was skeptical as well. i removed